docs/FAQ.rst

```diff
@@ -0,0 +1,17 @@
+Frequently Asked Questions
+==========================
+
+1. Why is my image "flipped" when I display it?
+
+At EuXFEL, the convention is to show detector images looking at the front of the detector, meaning the beam coming from behind you. You can get the other view with EXtra-geom's plotting methods by passing `frontview=False`.
+
+If you use matplotlib's `imshow(array)` to display an image, it may be upside down. You can correct this by passing `imshow(array, origin='lower')`.
+
+2. What are "slow_scan" and "fast_scan"?
+
+They are directions, or axes, in the HDF5 files. They are not always the same for different detectors. For example:
+
+- In LPD, "slow_scan" corresponds to the y-axis and "fast_scan" to the x-axis.
+- In AGIPD, "slow_scan" is the y-axis, whereas "fast_scan" is the y-axis
```

Comment:
I think we should try to explain what they mean and why we're not using x & y, rather than making a list of specific detectors. It's not as simple as this suggests - e.g. half the AGIPD-1M modules are 'upside down', so fast scan might be +y or -y depending on which module you look at. There's even an idea of using AGIPD in a 'windmill' arrangement, where fast scan would be x for two quadrants and y for two. I don't know if that will ever happen, but CSPAD (at LCLS) really does have modules in different orientations.

When you index the data in Python or C, fast_scan will be the last index, and slow_scan the one before that: `arr[..., slow_scan, fast_scan]`. (Fortran indexing works the other way round, but we probably don't need to mention that). The fast scan axis is the innermost one, where a row of pixels along that axis is stored together.

We talk about slow-scan and fast-scan rather than x & y, because the axes of the data array don't always represent the same physical directions. Working along the fast-scan axis might be increasing y in one AGIPD module, and decreasing y in another.

We could also point people to the `data_coords_to_positions()` method, which converts (module_no, slow_scan, fast_scan) data coordinates to (x, y, z) positions.